.circleci/config.yml

```diff
@@ -3,120 +3,25 @@ version: 2.1
-    - run:
-        name: Configure test env name
-        command: echo 'export INSTANCE_NAME="ranchhand-$CIRCLE_WORKFLOW_JOB_ID-$CIRCLE_BUILD_NUM"' >> $BASH_ENV
-    - run:
-        name: Create test env
-        command: |
-          INSTANCE_BLUEPRINT_ID=<< parameters.blueprint_id >> \
-          INSTANCE_TAGS="key=Repo,value=$CIRCLE_REPOSITORY_URL key=Branch,value=$CIRCLE_BRANCH key=Commit,value=$CIRCLE_SHA1 key=BuildUrl,value=$CIRCLE_BUILD_URL" \
-          test/e2e/run.sh setup
-    - run:
-        name: Execute ranchhand
-        command: |
-          ./ranchhand run \
-            --node-ips $(cat instance-ip) \
-            --ssh-user << parameters.ssh_user >> \
-            --ssh-key-path ~/.ssh/id_rsa_5a19aee7ef984f2f68c3be7262f91d35 \
-            --ssh-connect-timeout 180 \
-            --admin-password "$3cr3t$auc3"
-    - run:
-        name: Teardown test env
-        when: always
-        command: test/e2e/run.sh teardown
+      - image: ubuntu:16.04
 
 jobs:
   build:
+    description: "Check Ansible Syntax"
     executor: default
     steps:
-    - checkout
-    - restore_cache:
-        keys:
-        - v1-gomod-cache-{{ checksum "go.sum" }}
-    - run:
-        name: Build project
-        command: make build
-    - save_cache:
-        key: v1-gomod-cache-{{ checksum "go.sum" }}
-        paths:
-        - /go/pkg/mod
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ranchhand
-  unittest:
-    executor: default
-    steps:
-    - checkout
-    - restore_cache:
-        keys:
-        - v1-gomod-cache-{{ checksum "go.sum" }}
-    - run: make test
-  e2e_ubuntu:
-    executor: default
-    steps:
-    - launch_e2e:
-        ssh_user: ubuntu
-        blueprint_id: ubuntu_16_04_2
-  e2e_centos:
-    executor: default
-    steps:
-    - launch_e2e:
-        ssh_user: centos
-        blueprint_id: centos_7_1901_01
-  release:
-    executor: default
-    steps:
-    - checkout
-    - run: curl -sL https://git.io/goreleaser | VERSION=${GORELEASER_VERSION} bash
-
-workflows:
-  version: 2
-  build_and_test:
-    jobs:
-    - build
-    - unittest:
-        requires:
-        - build
-    - e2e_ubuntu:
-        requires:
-        - unittest
-    - e2e_centos:
-        requires:
-        - unittest
-  release:
-    jobs:
-    - release:
-        filters:
-          branches:
-            ignore: /.*/
-          tags:
-            only: /v[0-9]+(\.[0-9]+)*(-.*)*/
+      - run: 
+          name: Install Ansible & Dependencies
+          command: |
+            apt-get update
+            apt-get install -y software-properties-common python-apt python3-pip libssl-dev
+            update-alternatives --install /usr/bin/python python /usr/bin/python3.5 2
+            update-alternatives  --set python /usr/bin/python3.5
+            pip3 install ansible
+      - checkout
+      - run: 
+          name: Check Ansible Syntax
+          command: ansible-playbook -i '127.0.0.1,' ansible/prod.yml --syntax-check
```

Comment:
This config already has a way to stand up both Ubuntu and CentOS for _real_ testing. That seems better than just linting for syntax errors.

Reply:
Agreed. Just got the e2e integration test working.